recipe/run_test.py

```diff
@@ -1,11 +1,57 @@
-import ogr
-
-cnt = ogr.GetDriverCount()
-for i in range(cnt):
-    print(ogr.GetDriver(i).GetName())
+from osgeo import gdal
+from osgeo import ogr
+from osgeo import osr
 
+# this module does some tests
 import os1_hw
 
+# Set GDAL_DATA. This is done normally done by the activate script,
+# but this doesn't happen in the testing environment
+import os
+if 'LIBRARY_PREFIX' in os.environ:
+    # Windows
+    gdalData = os.path.join(os.environ['LIBRARY_PREFIX'], 'share', 'gdal')
+else:
+    # Linux/OSX
+    gdalData = os.path.join(os.environ['PREFIX'], 'share', 'gdal')
+
+os.environ['GDAL_DATA'] = gdalData
```

Comment:
We should patch our `gdal` to know about path without depending on a environment variable. So thing "just work."

(But that is out of the scope of this PR.)